tools/generator/SourceWriters/MethodCallback.cs

```diff
@@ -56,6 +58,15 @@ public MethodCallback (GenBase type, Method method, CodeGenerationOptions option
 
 		protected override void WriteBody (CodeWriter writer)
 		{
+			writer.WriteLine ("if (!global::Java.Interop.JniEnvironment.BeginMarshalMethod (jnienv, out var __envp, out var __r)) ");
```

Comment:
is there a reason this line ends with a space?  It causes `git show`/`git diff` to "light up red" for the space before newline:

<img width="909" alt="image" src="https://github.com/user-attachments/assets/de869e7e-de21-4013-bf38-81c4d9ddc216">